home/templates/blocks/media.html

```diff
@@ -1,23 +1 @@
-{% load wagtailcore_tags i18n %}
-
-{% if value.type == 'video' %}
-    <div>
-        <video preload="none" width="320" height="240" controls>
-            {% for source in value.sources %}
-                <source src="{{ source.src }}" type="{{ source.type }}">
-            {% endfor %}
-            {% translate "Your browser does not support video playback." %}
-        </video>
-    </div>
-    <p class='article__content--video'>{% blocktranslate %}If you cannot view the above video, you can instead {{ start_link }}download it{{ end_link }}.{% endblocktranslate %}</p>
-{% else %}
-    <div>
-        <audio preload="none" controls>
-            {% for source in value.sources %}
-                <source src="{{ source.src }}" type="{{ source.type }}">
-            {% endfor %}
-            {% translate "Your browser does not support audio playback." %}
-        </audio>
-    </div>
-    <p class='article__content--audio'>{% blocktranslate %}If you cannot listen to the above audio, you can instead {{ start_link }}download it{{ end_link }}.{% endblocktranslate %}</p>
-{% endif %}
+{% extends template %}
```

Comment:
Is this file still required, or can it be removed completely?

Reply:
It's required and renders whichever template is specified in context.